api/tests/prescription/campaign-participation/unit/domain/models/CampaignParticipation_test.js

```diff
@@ -192,8 +192,8 @@ describe('Unit | Domain | Models | CampaignParticipation', function () {
       });
     });
 
-    context('when the campaign is archived', function () {
-      it('throws an ArchivedCampaignError error', async function () {
+    context('when the campaign is not accessible', function () {
+      it('throws an ArchivedCampaignError error when archived', async function () {
         const campaign = domainBuilder.buildCampaign({ archivedAt: new Date() });
```

Comment:
**suggestion :** { isAccessbile: () => false } au lieu de passer un domain builder qui execute la logique du model.

Reply:
ne faire qu'un seul it qui throw